haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc

```diff
@@ -69,6 +69,16 @@ function haskala_book_book_fields_content_type_render($subtype, $conf, $args, $c
       if ($wrapper->$field_name->value()) {
         $field = render($node_view[$field_name]);
         $tabs_fields[$tab->label][] = $field;
+
+        // Role of preface writer
+        if ($field_name == "field_writer_of_preface") {
+          $production_id = haskala_book_prefaces_tab_additional_info($node->nid, $wrapper->$field_name->value()->nid);
+          if (isset($production_id)) {
```

Comment:
$production_id is always set at this point (Also if it's NULL). You probably want to check ``` if ($production_id) {```